src/M6Web/Bundle/LogBridgeBundle/Matcher/Status/Type/CompleteType.php

```diff
@@ -26,7 +28,7 @@ protected function getPattern()
      */
     protected function transform($config)
```

Comment:
suggestion (non-blocking): we can add type-hints on parameters and on return type ? (anywhere)
```suggestion
    protected function transform(string $config): array
```

Reply:
since this will be a new major release 
=> I agree that we should add all missing type-hints in this PR (in order to make a new major release for each type-hints added)

Reply:
It's done :)